### ISSUE TITLE
typos in doc/core/man/tap2{deb,rpm}.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2848]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2848|
|Type|defect|
|Created|2007-10-10 13:47:19Z|

package and explanation are both misspelled in each of these man pages

Attachments:

* [tap2xxx.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1ba/1ba65bc81ee7d284e60782ecd3308ec517b73b82/1654ec28531cc58a9134313954ecf35981561f4e.diff) (1665 bytes) - added by exarkun on 2007-10-10 13:48:13Z - fixes from downstream

<details><summary>Searchable metadata</summary>

```
trac-id__2848 2848
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1192024039000000 1192024039000000
changetime__1192024595000000 1192024595000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

(In [21343]) Apply patch fixing spelling errors in man pages

Author: A. Costa
Reviewer: exarkun
Fixes [#2848](https://github.com/twisted/twisted/issues/2848)

Fix the spelling of a couple words in the man pages for
tap2deb and tap2rpm.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
